viz/DepthMapVisualization.cpp

```diff
@@ -113,7 +124,7 @@ void DepthMapVisualization::updateMainNode ( osg::Node* node )
     }
     else
     {
-        colors->push_back(osg::Vec4(1.0,0,0.3,0.8));
+        colors->push_back(default_feature_color);
```

Comment:
Wouldn't it make more sense to keep the default feature color to its current value ? Not sure if/what you discussed with the primary users of this visualization ...